software/react-native/operations.md

```diff
@@ -2,8 +2,39 @@
 
 #### Deployment Strategy - Paid (Preferred)
```

Comment:
could potentially add in something about Flow or Typescript later on here

but one other thought would be: should we talk about linters? I think having an ES Lint as standard is probably good practice for our future projects